include/oneapi/dpl/experimental/kt/esimd_radix_sort_utils.h

```diff
@@ -489,32 +489,75 @@ VectorStore(uint32_t offset,
     return VectorStore<T, VSize, LANES>({offset, LaneStride*sizeof(T)}, data, mask);
 }
 
-template <typename T, int N>
-inline std::enable_if_t< (N*sizeof(T)/sizeof(uint32_t)<=64), __ESIMD_NS::simd<T, N> > BlockLoad(uint32_t slm_offset)
+template <int NElts>
+constexpr int
+CalculateApropriateDataSize()
```

Comment:
The solution looks good. 

However, have a couple of requests/questions regarding the design and names to improve readability:

```c++
// Create an alias to a type used for LSC block operations
// aligned because LSC operations require at least 4-byte alignment in memory 
using lsc_op_aligned_t = ::std::uint32_t;

// Rename CalculateApropriateDataSize
template<typename N>
constexp int lsc_op_block_size_rounding() {...}

// Rename EvalItemsCountIn
template <int N, typename T>
constexpr int lsc_op_block_size = N * sizeof(T) / sizeof(lsc_op_aligned_t);
```

What do you think about it?

Reply:
Renamed: ```CalculateApropriateDataSize``` -> ```lsc_op_block_size_rounding```

Reply:
Renamed: ```EvalItemsCountIn``` -> ```lsc_op_block_size```

Reply:
// Create an alias to a type used for LSC block operations
// aligned because LSC operations require at least 4-byte alignment in memory 
template<typename T>
using lsc_op_aligned_t = sizeof(T)<=4?::std::uint32_t : ::std::uint64_t;

Reply:
```lsc_op_aligned_t``` - implemented, but required to compile and check errors in test

Reply:
Checked: test passed.